ui/app/helpers/utils/i18n-helper.js

```diff
@@ -108,9 +108,44 @@ export async function fetchLocale(localeCode) {
   }
 }
 
+const numberFormatLocaleData = new Set();
+
+async function loadNumberFormatLocaleData(localeCode) {
+  const languageTag = localeCode.split('_')[0];
+  if (
+    Intl.NumberFormat &&
+    typeof Intl.NumberFormat.__addLocaleData === 'function' &&
+    !numberFormatLocaleData.has(languageTag)
+  ) {
+    const localeDataScript = await fetchNumberFormatLocaleData(languageTag);
+    if (!localeDataScript) {
+      return;
+    }
+    const localeDataRegex = `\\/\\* @generated \\*\\/
```

Comment:
Wow O_O. I think I see why you did this, but this is pretty unpleasant. I'm going to see if I can get a dynamic require working here instead.

Reply:
Done in bd58656ea96d8c87ca59448331eec5af77e1b292 - let me know what you think!